users/forms.py

```diff
@@ -0,0 +1,11 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm
+
+
+class UserRegisterationForm(UserCreationForm):
```

Comment:
sorry to bother but its spelled 'registration' without an 'e' instead of registeration

Reply:
I think the correct spelling it is 'registration' :)

Reply:
No problem:)
Changed to 'UserRegistrationForm'